virttest/virt_vm.py

```diff
@@ -1217,8 +1222,11 @@ def print_guest_network_info():
                     break
                 raise
             except Exception as err:
+                time.sleep(0.5)
```

Comment:
Hmm, where have I seen this... would be nice to give me some credit for it and include me in the `Signed-off` :wink:

Reply:
@ldoktor Haha, I did so https://github.com/avocado-framework/avocado-vt/pull/2488#issuecomment-602864040